experimental/rule_inference/README.md

```diff
@@ -0,0 +1,52 @@
+# PiranhaAgent
+
+PiranhaAgent uses OpenAI's GPT-4 model to infer piranha rules from examples. 
+It generates these rules in TOML format, which can be applied to refactor other parts of the codebase.
+
+## Requirements
+
+The following are the prerequisites to use PiranhaAgent:
+
+- Python 3.7 or higher
+- OpenAI Python library
+- Tree-sitter Python library
+- Tree-sitter-languages Python library
+## Install 
```

Comment:
Ow. I meant you have to fill in the holes and put text like that

Reply:
can we have a simple bash snippet to "install" this tooling?

Reply:
ah I see! like install.sh?